tests/php/test-class-amp-gfycat-embed-handler.php

```diff
@@ -5,21 +5,25 @@
  * @package AMP.
  */
 
+use AmpProject\AmpWP\Tests\WithoutBlockPreRendering;
+
 /**
  * Class AMP_Gfycat_Embed_Handler_Test
  *
  * @covers AMP_Gfycat_Embed_Handler
  */
 class AMP_Gfycat_Embed_Handler_Test extends WP_UnitTestCase {
 
+	use WithoutBlockPreRendering {
+		setUp as public prevent_block_pre_render;
+	}
+
 	/**
 	 * Set up.
-	 *
-	 * @global WP_Post $post
 	 */
 	public function setUp() {
-		global $post;
 		parent::setUp();
+		$this->prevent_block_pre_render();
```

Comment:
This calls `parent::setUp()` twice, once here, and once in `WithoutBlockPreRendering ::setUp` (which is mapped to `$this->prevent_block_pre_render();` here).